### PR TITLE
Cache elasticsearch-6.6.0.deb

### PR DESCRIPTION
**Pull request by cotsog**
Travis CI recipe to cache elasticsearch-6.6.0.deb to make builds more resilient to network connectivity issues.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,9 +15,23 @@ services:
...
+    - ELASTICSEARCH_DEB_PACKAGE=elasticsearch-${ELASTICSEARCH_DEB_VERSION}.deb
 
 before_install:
-  - curl -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.6.0.deb && sudo dpkg -i --force-confnew elasticsearch-6.6.0.deb && sudo service elasticsearch restart
+  - |
+    if [ ! -f ./deb-cache/$ELASTICSEARCH_DEB_PACKAGE ]
+    then
+      curl -o ./deb-cache/$ELASTICSEARCH_DEB_PACKAGE https://artifacts.elastic.co/downloads/elasticsearch/$ELASTICSEARCH_DEB_PACKAGE
+    fi
+  - sudo dpkg -i --force-confnew ./deb-cache/elasticsearch-6.6.0.deb && sudo service elasticsearch restart
```
Comment (kuisathaverat) [inline .travis.yml]:
```suggestion
  - sudo dpkg -i --force-confnew ./deb-cache/$ELASTICSEARCH_DEB_PACKAGE && sudo service elasticsearch restart
```

Comment (cotsog) [reply .travis.yml]: Thanks for the catch!

**Comment by Qard** [comment]
Would you mind squashing with the commit title `ci(travis): cache elasticsearch deb`? That will allow this to pass the linter and run the next stage of tests.

Other than that, LGTM.

**Comment by cotsog** [comment]
I'll open another PR.

**Comment by cotsog** [comment]
New PR: https://github.com/elastic/apm-agent-nodejs/pull/1247

**Pull request closed without merging**